src/axes/axis.ts

```diff
@@ -105,6 +105,27 @@ export class Axis<D> extends Component {
     this._scale.offUpdate(this._rescaleCallback);
   }
 
+  /**
+   * Gets the tick label data on a element.
+   *
+   * @param {Element} element
+   */
+  public tickLabelDataOnElement(element: Element) {
+    if (element === null || element === undefined) { return undefined; }
```

Comment:
nit: you can do `== null` to catch both null and undefined (this is the only accepted use of ==)